src/WorkOS.net/Services/UserManagement/_interfaces/VerifyEmailOptions.cs

```diff
@@ -0,0 +1,22 @@
+namespace WorkOS
+{
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// The parameters to verify user email using a verification token.
+    /// </summary>
+    public class VerifyEmailOptions : BaseOptions
+    {
+        /// <summary>
+        /// The verification code emailed to the user.
+        /// </summary>
+        [JsonProperty("user_id")]
```

Comment:
Should we be excluding this field from the JSON since it is passed in the URL?

Reply:
Thinking it might be better to give this one the same [JsonIgnore] treatment and just only pass in VerifyEmailOptions to VerifyEmail()

Reply:
Changed in [4ccbb01](https://github.com/workos/workos-dotnet/pull/148/commits/4ccbb01ab6f5e3321ccf5f3b1840060a8e793c95)